alma/lib/Utils/Settings.php

```diff
@@ -140,28 +140,73 @@ public static function canLog()
         return (bool) (int) self::get('ALMA_ACTIVATE_LOGGING', false);
     }
 
+    /**
+     * Get value ALMA_ACTIVATE_SHARE_OF_CHECKOUT
+     *
+     * @return string
+     */
     public static function canShareOfCheckout()
     {
-        return boolval(self::get(ShareOfCheckoutAdminFormBuilder::ALMA_ACTIVATE_SHARE_OF_CHECKOUT, false));
+        return self::get(ShareOfCheckoutAdminFormBuilder::ALMA_ACTIVATE_SHARE_OF_CHECKOUT, ShareOfCheckoutAdminFormBuilder::ALMA_SHARE_OF_CHECKOUT_CONSENT_NO_ANSWER);
```

Comment:
```suggestion
        return self::get(ShareOfCheckoutAdminFormBuilder::ALMA_SHARE_OF_CHECKOUT_STATE, ShareOfCheckoutAdminFormBuilder::ALMA_SHARE_OF_CHECKOUT_CONSENT_NO_ANSWER);
```
As suggested above for the `canShareOfCheckout` method that is no longer a `bool`,
maybe the constant & settings name could become something like `ALMA_SHARE_OF_CHECKOUT_STATE`